src/primitives/cs_helpers.rs

```diff
@@ -106,16 +106,16 @@ pub fn constraint_evaluation_over_lde(
         lde_size,
     );
 
-    let _ = boojum_cuda::gates::evaluate_gates(
-        &gates,
-        &variable_columns_matrix,
-        &witness_columns_matrix,
-        &constant_columns_matrix,
-        challenge,
-        &mut quotient_matrix,
-        0,
-        get_stream(),
-    )?;
-
-    Ok(())
+    if_not_dry_run! {
```

Comment:
i think this whole `constraint_evaluation_over_lde` is dead code which you can delete if you want.

Reply:
i will keep that in mind if i ever touch the code again